enterprise/server/auditlog/auditlog.go

```diff
@@ -211,6 +197,25 @@ func cleanRequest(e *alpb.Entry_ResourceRequest) *alpb.Entry_ResourceRequest {
 	return e
 }
 
+func (l *Logger) enrichRequest(ctx context.Context, e *alpb.Entry_ResourceRequest) (*alpb.Entry_ResourceRequest, error) {
+	e = proto.Clone(e).(*alpb.Entry_ResourceRequest)
+	if r := e.UpdateGroupUsers; r != nil {
+		for _, u := range r.Update {
+			uid := u.GetUserId().GetId()
+			userData, err := l.env.GetUserDB().GetUserByID(ctx, uid)
+			if err != nil {
+				return nil, err
+			}
+			if userData.Email != "" {
+				u.UserId.Id += " (" + userData.Email + ")"
+			} else if userData.FirstName != "" && userData.LastName != "" {
+				u.UserId.Id += " (" + userData.FirstName + " " + userData.LastName + ")"
```

Comment:
Are you sure you want to do the "(" appending here or is that a client side display thing?

Reply:
test